dev/com.ibm.ws.webcontainer.servlet.3.1_fat.2/test-applications/FormLogin_ReadListener.war/resources/logout.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<!--
+    Copyright (c) 2014 IBM Corporation and others.
```

Comment:
update CC ?

Reply:
This is an existing resource that was copied over to the new bucket: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.3.1_fat/test-applications/FormLogin_ReadListener.war/resources/WEB-INF/web.xml